tutor/resources/styles/components/notes/summary-page.scss

```diff
@@ -2,11 +2,34 @@
   // Force fixed modal-scroll-top button to attach to modal
   // by using a no-effect transform.
   transform: rotate(0);
+  // make the modal biggers
+  @media screen and ( max-width: $mobile-collapse-breakpoint ){
+    margin: 1% auto;
+    width: 95%;
```

Comment:
Could do something like `calc(100vw - #{$mobile-margin} * 2);` if you wanted to keep the modal aligned and covering the content container below

Reply:
yes, using % is probably fine since this is inside the mobile breakpoint and 1% won't ever grow to large.

Reply:
thanks nathan 👍